lua/ttt2/libraries/orm.lua

```diff
@@ -0,0 +1,168 @@
+---
+-- Object Relational Model
+-- @author Histalek
+
+--local baseclass = baseclass
+local sql = sql
+
+orm = orm or {}
+
+local ormodel = {}
+
+local function SetupModelFunctions(tableName, primaryKey, dataStructure)
+
+	local model = {}
+	local sanTableName = sql.SQLIdent(tableName)
+
+	model.New = function(self, data)
+		local object = data or {}
+
+		-- TODO needed?
```

Comment:
Todo?

Reply:
Well it's one of the two things why this PR is a draft :P